pkg/fetch/caching_fetcher.go

```diff
@@ -39,10 +38,7 @@ func (cf *cachingFetcher) FetchBlob(ctx context.Context, req *remoteasset.FetchB
 
 	var oldestContentAccepted time.Time = time.Unix(0, 0)
 	if req.OldestContentAccepted != nil {
-		oldestContentAccepted, err = ptypes.Timestamp(req.OldestContentAccepted)
-		if err != nil {
-			return nil, err
-		}
+		oldestContentAccepted = req.OldestContentAccepted.AsTime()
```

Comment:
Maybe good to still keep a `req.OldestContentAccepted.CheckValid()` here?

Reply:
Done!